src/workerd/io/compatibility-date.capnp

```diff
@@ -721,4 +721,10 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
       $impliedByAfterDate(name = "nodeJsCompat", date = "2025-04-01");
   # Automatically populate process.env from text bindings only
   # when nodejs_compat is being used.
+
+  importableEnv @77 :Bool
+      $compatEnableFlag("allow_importable_env")
```

Comment:
Since this isn't a breaking change, perhaps the meanings of enable/disable should be reversed here. The flag would exist _only_ for people like me to block populating of env. But for everyone else, it's exposed by default, even for past compat dates.

Reply:
Done.